ui/src/vayla-design-lib/dropdown/dropdown.tsx

```diff
@@ -84,16 +84,16 @@ export const Dropdown = function <TItemValue>({
         }
         earlySelect.current = false;
     });
-    const [open, setOpen] = React.useState(
-        props.openOverride !== undefined ? props.openOverride : false,
-    );
+    const [open, setOpen] = React.useState(false);
```

Comment:
Eriytetty `open` kokonaan overridesta. Tätä myötä kun `openOverride` asetetaan `undefined`-asentoon, voidaan alta kaivaa wanha `open`:in arvo aiempaa paljon helpommin. Aiempi tapa aiheutti ongelmia silloin kun `openOverride` vaihtui kesken matkan